app/battle.py

```diff
@@ -0,0 +1,24 @@
+
+from typing import List, Union
+
+from app.knight import Knight
+
+
+def perform_battle(knights: List[Knight]) -> Union[str, None]:
+    lancelot = knights[0]
+    mordred = knights[2]

```

Comment:
Not true. If I will change the order of knights in json your code stop working. They must fight by name